### PR TITLE
Code changes requested in https://github.com/dotnet/docs/pull/9776

### PR DESCRIPTION
**Pull request by v-thepet**
## Summary

Changes requested to the code samples in how-to-enumerate-directories-and-files.md. Content refresh and requested content changes are at https://github.com/dotnet/docs/pull/9776. 
Code changes also requested there and implemented here:

(../../../samples/snippets/csharp/VS_Snippets_CLR_System/system.io.directory.enumeratefiles/cs/program.cs#1)]

in the VB example, there should be a blank line between the last Imports statement and the Module statement.
the exception names should use camel casing or just be ex
the 2 Console.WriteLine statements with format strings should be replaced with interpolated strings..

(../../../samples/snippets/csharp/VS_Snippets_CLR_System/system.io.enumdirs1/cs/program.cs#1)]
delete using System.Linq;
exception names should use camel casing. We generally just name an exception variable ex.
we've been replaying format strings with interpolated strings; so:
$"{0}", dir.Substring(dir.LastIndexOf("\\") + 1) --> "{dir.Substring(dir.LastIndexOf("\\") + 1)}"
"{0} directories found.", dirs.Count --> $"{dirs.Count} directories found."

(../../../samples/snippets/csharp/VS_Snippets_CLR_System/system.io.directoryinfo.enumdirs/cs/program.cs)]

In the C# example, only the first two using statements are needed.
The VB example should have a full example that corresponds to the C# example, not just a fragment
indentation in the C# example should be fixed.
See if the Snippet1 tag is used elsewhere in the dotnet/docs or dotnet/dotnet-api-docs repos (search for "directoryinfo.enumdirs#1"). If it's not, delete it.

(../../../samples/snippets/csharp/VS_Snippets_CLR_System/system.io.directoryinfo.enumeratedirectories/cs/program.cs#1)]

use interpolated strings except for the VB line that includes vbTab twice. That's best as a format string, with vbTab as one of the format items.
in the VB example, delete Imports System
exception variable names should be changed (camel cased or just ex)

Part of User Story https://mseng.visualstudio.com/TechnicalContent/_queries/edit/1374049/?triage=true

### CODE REVIEW

File: snippets/csharp/VS_Snippets_CLR_System/system.io.enumdirs1/cs/program.cs [hunk 1]
```diff
@@ -17,9 +16,9 @@ private static void Main(string[] args)
                     
             foreach (var dir in dirs)
             {
-                Console.WriteLine("{0}", dir.Substring(dir.LastIndexOf("\\") + 1));
+                Console.WriteLine($"{dir.Substring(dir.LastIndexOf("\\") + 1)}");
```
[the remark this answers is not in the document]
Comment (v-thepet) [reply snippets/csharp/VS_Snippets_CLR_System/system.io.enumdirs1/cs/program.cs]: @BillWagner - This code runs fine in VS Code on my Ubuntu 18.04 box actually?

[the remark this answers is not in the document]
Comment (BillWagner) [reply snippets/csharp/VS_Snippets_CLR_System/system.io.enumdirs1/cs/program.cs]: @v-thepet Yes, it runs, but the output is very different.  On Windows, it prints the directories in your Documents folder, relative to the documents folder.

On my Mac, it prints the full path to every directory in my home folder.  

Try changing the "\\" to "/" to see what I mean.

File: snippets/visualbasic/VS_Snippets_CLR_System/system.io.directoryinfo.enumeratedirectories/vb/program.vb [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
         Try
             For Each fi In diTop.EnumerateFiles()
                 Try
                     ' Display each file over 10 MB;
                     If fi.Length > 10000000 Then
                         Console.WriteLine("{0}" & vbTab & vbTab & "{1}", fi.FullName, fi.Length.ToString("N0"))
                     End If
-                Catch UnAuthTop As UnauthorizedAccessException
-                    Console.WriteLine("{0}", UnAuthTop.Message)
+                Catch unAuthTop As UnauthorizedAccessException
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_CLR_System/system.io.directoryinfo.enumeratedirectories/vb/program.vb]: This is a nit, but typically exception variables aren't given meaningful names -- they're just named `e` or `ex`.

**Comment by v-thepet** [comment]
@BillWagner @rpetrusha - Corrections are made, please review.

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @v-thepet. I'll merge your PR now.

**Pull request merged**